### PR TITLE
Removed systemd dependencies

### PR DESCRIPTION
**Pull request by stefonarch**

### CODE REVIEW

File: startlxqtwayland.in [hunk 1]
```diff
@@ -95,11 +95,11 @@ if [ $COMPOSITOR = "labwc" ]; then
     fi
 
 # enable cursor on VM
-    VIRT=$(systemd-detect-virt)
-    if [ ! "$VIRT" = none ]; then
+    if [ $(hostnamectl | grep Virtualization) ];then
```
Comment (palinek) [inline startlxqtwayland.in:98]: Despite the commit log message, here you're using systemd's component

Comment (tsujan) [reply startlxqtwayland.in:98]: > What about using cat /sys/devices/virtual/dmi/id/product_family ?

I don't think it's reliable for this kind of usage.

> We could also drop this condition

Why dropping it totally? Why not dropping it only in the exceptional case where `systemd` doesn't exist?

I don't think there's a tool as handy as `systemd-detect-virt`.

File: startlxqtwayland.in [hunk 2]
```diff
@@ -94,16 +94,19 @@ if [ $COMPOSITOR = "labwc" ]; then
        fi
     fi
 
-# enable cursor on VM
-    VIRT=$(systemd-detect-virt)
-    if [ ! "$VIRT" = none ]; then
-        export WLR_NO_HARDWARE_CURSORS=1
-        echo "Running on virtualized hardware"
+    # enable cursor on VM (systemd only)
+    PID1=$(ps -p 1 -o comm=)
```
Comment (palinek) [inline startlxqtwayland.in:98]: Seems a bit overcomplicated here for me. How about something like:
```sh
if type systemd-detect-virt > /dev/null 2>&1 && [ "$(systemd-detect-virt)" != 'none' ]; then
    export WLR_NO_HARDWARE_CURSORS=1
    echo "Running on virtualized hardware"
fi
```

**Comment by tsujan** [comment]
It can be merged now; can't it?

**Comment by stefonarch** [comment]
> It can be merged now; can't it?

I think so, tests were successful.

**Pull request merged**